parking-lot-help-request/README.md

```diff
@@ -0,0 +1,118 @@
+# Parking Lot Help Request
+
+Host-free low latency notification system to alert attendants in a parking lot or facility that customers need assistance. To run this project youself you'll need to:
+
+* [Purchase the necessary hardware and configure it](#hardware).
+* [Configure Notehub](#notehub).
+* [Configure the project's Notecard](#notecard-firmware).
+* [Set up a Twilio account and configure the data to route from Notehub to Twilio](#twilio-firmware).
+## Hardware
+
+The following hardware is required to run the Parking Lot Help Request application.
+
+* [Notecarrier A with pre-soldered headers](https://shop.blues.io/products/carr-al)
+* [Notecard LTE Cat-1](https://shop.blues.io/products/note-wbna-500) (for use in North America)
```

Comment:
Not sure we want to make this NA only. Maybe we could just say that you need a cellular Notecard, and say that Cat-1 is highly recommended?

Reply:
Very good suggestion. Thanks